tabulon_dxf/src/lib.rs

```diff
@@ -440,11 +445,7 @@ pub fn load_file_default_layers(path: impl AsRef<Path>) -> DxfResult<TDDrawing>
         let eh = EntityHandle(NonZeroU64::new(e.common.handle.0).unwrap());
         let lh = handle_for_layer_name[e.common.layer.as_str()];
```

Comment:
You look up the layer handle here by name. But that's the only use of `handle_for_layer_name` ... and then you look up the layer  below with that being the only use of `layers` ...

So I'm wondering if this is doing some extra work / storage or not?